test/parser/test.ml

```diff
@@ -857,7 +853,7 @@ let () =
       let actual =
         let {permissive; sections_allowed; location; parser_input; _} = case in
 
-        let dummy_filename = "f.ml" in
+        let dummy_filename = "f.ml" in (* FIXME: the .ml extension is not needed here. *)
```

Comment:
Why change it?

Reply:
First of all, it's incorrect to have the extension. The root represents a page and not a compilation unit. In the error reporting I also assume the filename does not have an extension and add the `.mld` extension to it when printing an error.

I didn't change this in this PR because it would add a lot of noise in many (all?) parser expect files.

Reply:
Actually the error reporting situation no longer applies as I deleted the requirement on pages having titles. I still think it should be changed.